### DISCUSSION TITLE
Set Timer for auto deletion of pads

### DISCUSSION THREAD

**Discussion by gk4468** [ether/etherpad-lite#5055] [Q&A]
Is it possible to set a timer that delete your pad after not being edited for a time. I didn't find any option in the documentation. 
If this is possible, could I set the timer to 180 days.

**Comment by rhansen** [comment]
[ep_delete_after_delay](https://www.npmjs.com/package/ep_delete_after_delay) appears to automatically delete pads after a certain amount of time regardless of how recently it was edited. Is that sufficient for your needs? If not, you could fork that plugin and tweak the logic to delete pads that have been idle for more than the configured threshold.
